scripts/_common.sh

```diff
@@ -24,7 +24,7 @@ log_file="${log_path}/pyinventory.log"
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies="build-essential python3-dev python3-pip python3-virtualenv git \
+pkg_dependencies="build-essential python3-dev python3-pip python3-venv python3-virtualenv git \
```

Comment:
Hm. Interesting.

Think only "python3-venv" is needed. Because i changed the venv creation here: https://github.com/YunoHost-Apps/pyinventory_ynh/commit/12739d04d725ec0c508294ecf01d2911f2966cc3#diff-6233345de4867c7ccf72bac1fc1067988d2a4c8e1ab9a746956eafccf88cfed2

I will try to test this again.

Reply:
Hm. I'm confused:

```
root@yunohost:~# python3
Python 3.7.3 (default, Jul 25 2020, 13:03:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import venv
>>> venv.__file__
'/usr/lib/python3.7/venv/__init__.py'
```
And `/usr/lib/python3.7/venv` is not included in both packages:

- https://packages.debian.org/buster/all/python3-virtualenv/filelist
- https://packages.debian.org/buster/amd64/python3-venv/filelist

isn't it?

The package venv is part of the official python distribution. But i think debian split python?!?

Reply:
None of these packages are needed?!?

```
root@yunohost:~# apt purge python3-venv python3-virtualenv
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  postgresql python3-crypto python3-entrypoints python3-keyring python3-keyrings.alt python3-ldap python3-pip python3-pyasn1 python3-pyasn1-modules python3-secretstorage python3-wheel python3-xdg
Use 'apt autoremove' to remove them.
The following packages will be REMOVED:
  pyinventory-ynh-deps* python3-venv* python3-virtualenv* virtualenv*
0 upgraded, 0 newly installed, 4 to remove and 0 not upgraded.
After this operation, 191 kB disk space will be freed.
Do you want to continue? [Y/n] y
(Reading database ... 43617 files and directories currently installed.)
Removing pyinventory-ynh-deps (0.8.1~ynh6) ...
Removing python3-venv (3.7.3-1) ...
Removing virtualenv (15.1.0+ds-2) ...
Removing python3-virtualenv (15.1.0+ds-2) ...
Processing triggers for man-db (2.8.5-2) ...
root@yunohost:~# python3 -m venv --help
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip] [--prompt PROMPT]
            ENV_DIR [ENV_DIR ...]
...
```

Reply:
I have tested with `python3-venv` added to `pkg_dependencies`
```
Package linter:                  SUCCESS
Installation:                    SUCCESS
Deleting:                        SUCCESS
Installation in a sub path:      SUCCESS
Deleting from a sub path:        SUCCESS
Installation on the root:        SUCCESS
Deleting from root:              SUCCESS
Upgrade:                         SUCCESS
Installation in private mode:    SUCCESS
Installation in public mode:     SUCCESS
Multi-instance installations:    SUCCESS
Malformed path:                  Not evaluated.
Port already used:               Not evaluated.
Backup:                          SUCCESS
Restore:                         FAIL
Change URL:                      FAIL
Actions and config-panel:        Not evaluated.
```